src/components/LoanCards/ListLoanCard.vue

```diff
@@ -1,13 +1,154 @@
 <template>
-	<div class="list-loan-card">
-		ListLoanCard Placeholder
+	<div class="list-loan-card row">
+		<div class="list-loan-card-desktop-column show-for-large large-4 xlarge-4 columns">
```

Comment:
This should be your first column which stacks at the smaller sizes rather than duplicates the content.